README.md

```diff
@@ -102,6 +102,10 @@ WebAuthn.configure do |config|
   # Relying Party name for display purposes
   config.rp_name = "Example Inc."
 
+  # Optional numerical hint, in milliseconds to wait for attestation
+  # and assertion. This may be overridden by the browser.
+  config.timeout = 120000
```

Comment:
I'm trying to put myself in the shoes of the person configuring the gem for the first time with little context of the standard, and my first impression is that the user will have a hard time understanding what this timeout is for. I specifically mean this for the configuration attribute in the configuration, not the hash of options we give the gem's user.

How about a diff name for the configuration field? `default_client_timeout` maybe? @grzuy you wrote the issue, maybe you could suggest another name?

Second, I would be just a little more verbose on the comment for it or at least add a reference to [doc option](https://www.w3.org/TR/webauthn/#dom-publickeycredentialcreationoptions-timeout)

Reply:
Thanks for the feedback @brauliomartinezlm. I agree with your comment. It might not be immediately obvious for someone who isn't familiar with the spec. 

I've updated `timeout` to be `client_timeout` and I've updated the description to be more verbose.

Reply:
@brauliomartinezlm You're right. Just `timeout` is too obscure here.

I think we need to be as explicit and obvious as possible.

I would go with

```ruby
WebAuthn.configuration.credential_options_timeout
```
or if we want to start to namespacing properly, eventually we could...

```ruby
WebAuthn.configuration.credential_options.timeout
```